dealers_choice.js

```diff
@@ -11,26 +11,313 @@ app.use(staticMiddleWare);
+        });
+
+        if (selectedMountainRanges.length === 0) allCheckBoxes.forEach(box => selectedMountainRanges.push(box.id));
+
+        let mountainFilterCriteria = {
+          ranges: selectedMountainRanges,
+          maxSkill: skillValue
+        }
+
+        let newURL = "";
+        mountainFilterCriteria.ranges.forEach(range => newURL += range+"&")
+        window.location.href = "/entries/ranges/"+newURL+skillValue;
+      });
+
+
+    </script>
   </html>
   `
+  response.send(homeHtml);
+})
+
+app.get( '/entries/:ID', (request, response) => {
+  console.log(request.params.ID);
+  const ID = request.params.ID;
+  const entry = postBank.find(ID);
+  const pageHtml = `
+    <!DOCTYPE html>
+    <html lang="en">
+      <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="/style.css" />
+        <title>Fourteeners ID: ${ID}</title>
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+        <link href="https://fonts.googleapis.com/css2?family=Permanent+Marker&display=swap" rel="stylesheet">
+      </head>
+      <body id="indiv-body">
+        <div id="nav">
+          <ul style="background-color: lightslategrey">
+            <li class="link" style="float:right"><a class="active" href="#about">About</a></li>
+            <li id="icon"><a href="/"><img src="/fourteeners-home-alt.png" alt="The number 14 in front of a mountain range drawing"></a></li>
+          </ul>
+        </div>
+        <div id="hero" style="background-image: url(${entry.photo})">
+          <h1 class="h3">${entry["Mountain Peak"].toUpperCase()}</h1>
+        </div>
+        <div class="container">
+          <div class="indiv-entry-container">
+            <p class="indiv-subheading"><strong>${entry["Mountain Peak"]}</strong></p>
+            <p>Range: <strong>${entry["Mountain Range"]}</strong></p>
+            <p>Elevation: <strong>${entry.Elevation_ft}</strong></p>
+            <ul>
+              <li>Difficulty: ${entry.DifficultyDescription}</li>
+              <li>Standard Route: ${entry["Standard Route"]}</li>
+              <li>Hiking Distance: ${entry.Distance_mi}</li>
+              <li>Elevation gain: ${entry["Elevation Gain_ft"]}</li>
+              <li>Traffic Low: ${entry["Traffic Low"]}</li>
+              <li>Traffic High: ${entry["Traffic High"]}</li>
+            </ul>
+          </div>
+        </div>
+      </body>
+    </html>
+  `
+  response.send(pageHtml);
+})
+
+app.get( '/entries/ranges/:RangeIDs', (request, response) => {
+  console.log(request.params.RangeIDs);
+  let rangeID = request.params.RangeIDs;
+  let rangeArr = [];
+  if (rangeID.includes("&")) {
+    rangeArr = rangeID.split("&");
+  }
+  else {
+    rangeAArr = [rangeID];
```

Comment:
there's a typo in here; why was this not caught by node? maybe it can never be hit